### ISSUE TITLE
Feature request: extrapolation of regional scoreboard values

### ISSUE THREAD

**Issue by ianmcorvidae** [jonatkins/ingress-intel-total-conversion#1028]
A frequently-asked question during play is roughly "if things stay about how they are, which team will win this cycle in the region?" A good way to represent this, I think, would be to show (on a checkbox selected, presumably) what the end-of-cycle score would look like with some sort of extrapolation for checkpoints that haven't yet happened.

As far as a method of extrapolation, my best idea would be to use the median values of past checkpoints (to exclude e.g. the effects of single-checkpoint mega field spikes as well as more general fluctuation). Another option would be to use the most recent checkpoint's values.

**Comment by ryantheleach** [comment]
It should be the median to match expectations against the ingame median line.

**Comment by lablua** [comment]
@ryantheleach: What makes you think the in game line is the median, when the in game score shown is the mean? 

I do agree that median and most recent checkpoint are both good extrapolations to look at, unfortunately it varies which one is more useful. For example, if a large field was up overnight near the beginning of a cycle, it can take awhile for the median to adjust to something reasonable.

**Comment by ryantheleach** [comment]
Yeah I screwed up, meant to say mean, have no idea what I was thinking at the time of posting.

**Comment by McBen** [comment]
here is my try: pull-request  https://github.com/jonatkins/ingress-intel-total-conversion/pull/1045
